### ISSUE TITLE
Transaction wait: timed out

### ISSUE THREAD

**Issue by roytman** [IBM/ovsdb-etcd#203]
A lot of errors like:

> I0608 23:56:44.161148      16 handler.go:44] Transact request from 172.18.0.2:42232, params [OVN_Northbound map[columns:[options] op:wait rows:[map[options:[map [[mac_prefix 6e:a4:2a] [max_tunid 16711680] [northd_internal_version 21.03.1-20.16.1-56.0] [svc_monitor_mac e2:8e:b4:01:75:3c]]]]] table:NB_Global timeout:0 until:== where:[[_uuid == [uuid 75e8e765-1841-4e56-ab81-e0d9fa8c7a08]]]] map[op:update row:map[up:true] table:Logical_Switch_Port where:[[_uuid == [uuid fe927c58-fc82-40d6-b1a2-26e430039018]]]] map[op:update row:map[options:[map [[mac_prefix 6e:a4:2a] [max_tunid 16711680] [northd_internal_version 21.03.1-20.16.1-56.0] [svc_monitor_mac e2:8e:b4:01:75:3c]]]] table:NB_Global where:[[_uuid == [uuid 75e8e765-1841-4e56-ab81-e0d9fa8c7a08]]]] map[op:update row:map[up:false] table:Logical_Switch_Port where:[[_uuid == [uuid ef62a747-f444-4dae-8040-63b63fac7c3a]]]] map[comment:ovn-northd op:comment]]
I0608 23:56:44.161290      16 transact.go:563] new transaction [with size 5]: {"dbname":"OVN_Northbound","operations":[{"op":"wait","table":"NB_Global","rows":[{"options":["map",[["mac_prefix","6e:a4:2a"],["max_tunid","16711680"],["northd_internal_version","21.03.1-20.16.1-56.0"],["svc_monitor_mac","e2:8e:b4:01:75:3c"]]]}],"columns":["options"],"timeout":0,"where":[["_uuid","==",["uuid","75e8e765-1841-4e56-ab81-e0d9fa8c7a08"]]],"until":"=="},{"op":"update","table":"Logical_Switch_Port","row":{"up":true},"where":[["_uuid","==",["uuid","fe927c58-fc82-40d6-b1a2-26e430039018"]]]},{"op":"update","table":"NB_Global","row":{"options":["map",[["mac_prefix","6e:a4:2a"],["max_tunid","16711680"],["northd_internal_version","21.03.1-20.16.1-56.0"],["svc_monitor_mac","e2:8e:b4:01:75:3c"]]]},"where":[["_uuid","==",["uuid","75e8e765-1841-4e56-ab81-e0d9fa8c7a08"]]]},{"op":"update","table":"Logical_Switch_Port","row":{"up":false},"where":[["_uuid","==",["uuid","ef62a747-f444-4dae-8040-63b63fac7c3a"]]]},{"op":"comment","comment":"ovn-northd"}]}
I0608 23:56:44.161672      16 transact.go:224] etcd transaction: #then 4, #events 0, #events-nil 0
I0608 23:56:44.161679      16 transact.go:230] etcd processing(0): #then 4, #events 0, #events-nil 0
E0608 23:56:44.164894      16 transact.go:1838] wait: timed out

**Comment by hunchback** [comment]
Another example:

```
I0609 16:02:09.232972      17 transact.go:1907] "checking row equal" txnid="63FQnPGF3D2cV7ZZWDDoLW" row=&map[_uuid:[uuid 739e1b8e-f56b-40f7-960a-81c8463148f6] connections:{GoSet:[]} external_ids:{GoMap:map[]} hv_cfg:0 hv_cfg_timestamp:0 ipsec:false name: nb_cfg:0 nb_cfg_timestamp:0 options:{GoMap:map[mac_prefix:0e:38:28 max_tunid:16711680 northd_internal_version:21.03.1-20.16.1-56.0 svc_monitor_mac:7a:77:ec:b9:d9:f3]} sb_cfg:0 sb_cfg_timestamp:0 ssl:{GoSet:[]}] expected=map[options:{GoMap:map[e2e_timestamp:1623254528 mac_prefix:0e:38:28 svc_monitor_mac:7a:77:ec:b9:d9:f3]}] result=false
E0609 16:02:09.233044      17 transact.go:1928] "timed out" err="timed out" txnid="63FQnPGF3D2cV7ZZWDDoLW"
```

**Comment by roytman** [comment]
I see this entry in the logs

**Comment by hunchback** [comment]
the `timed out` messages only exist in the ovn-nb.

```
[root@ovn-control-plane ovn]# cat ovn-northd.log | grep 'timed out' | grep :6642 | head -n 1
[root@ovn-control-plane ovn]# cat ovn-northd.log | grep 'timed out' | grep :6641 | head -n 1
2021-06-09T16:02:09.245Z|21944|jsonrpc|DBG|tcp:172.18.0.3:6641: received error, error={"error":"timed out"}, id=3366
[root@ovn-control-plane ovn]#
```

**Comment by roytman** [comment]
Fixed by #255
